bazel/api-gen/generate_api_docs.bzl

```diff
@@ -15,7 +15,7 @@ def generate_api_docs(name, module_name, entry_point, srcs, import_map = {}, ext
         extra_entries = extra_entries,
     )
 
-    render_api_to_html(
```

Comment:
I might need help on this one. 

Those tests fail with 

`Error [ERR_REQUIRE_ESM]: require() of ES Module /private/var/tmp/......./bin/bazel/api-gen/rendering/render_api_to_html.sh.runfiles/npm/node_modules/shiki/dist/index.mjs not supported.
Instead change the require of /private/var/tmp/...../bin/bazel/api-gen/rendering/render_api_to_html.sh.runfiles/npm/node_modules/shiki/dist/index.mjs to a dynamic import() which is available in all CommonJS modules.`